aurora-postgresql-plugin/widgets/AuroraPostgres-batchsink.json

```diff
@@ -95,6 +124,20 @@
     }
   ],
   "outputs": [],
+  "filters": [
```

Comment:
remove this filter condition and let relationTableKey field always visible. Put a validation if user has selected upsert or update, then this field can not be null.
The reason for removing this filter is if user selects operation name as macro, this field will not be visible to set it as macro.

Reply:
done